tools/perf/components/perf_test_utils.py

```diff
@@ -43,16 +43,17 @@ def GetProcessOutput(args: List[str],
       while True:
         assert process.stdout is not None
         line = process.stdout.readline()
-        output += line
-        logging.debug(line.rstrip())
-        if process.poll() is not None:
```

Comment:
The problem was skipping the last lines of the output.
It's only for debug mode, but we currently use it for CI by default.